src/neuroconv/basedatainterface.py

```diff
@@ -92,13 +134,22 @@ def run_conversion(
         overwrite : bool, default: False
             Whether to overwrite the NWBFile if one exists at the nwbfile_path.
             The default is False (append mode).
+        backend : "hdf5" or a HDF5BackendConfiguration, default: "hdf5"
+            The type of backend to use when writing the file.
```

Comment:
This typing does not match line 118

Reply:
Good eye

https://github.com/catalystneuro/neuroconv/pull/822